perllib/FixMyStreet/App/Form/Page/Waste.pm

```diff
@@ -6,11 +6,21 @@ extends 'FixMyStreet::App::Form::Page::Simple';
 has title => ( is => 'ro', 'isa' => 'Str', lazy => 1, builder => '_build_title' );
 # So we can insert cobrand specific service name
 has title_ggw => ( is => 'ro', isa => 'Str' );
+has bulky_general_title => ( is => 'ro', 'isa' => 'Str' );
```

Comment:
bulky_general_title isn't actually used, it looks like?

Reply:
It's used in perllib/FixMyStreet/App/Form/Waste/Bulky.pm which is how I thought it took effect though I was kind of muddling through

Reply:
Where is it used? I see you've added some blank entries for it, but they're not actually then used anywhere?

Reply:
Yeah, was blundering about in the dark a bit there!